### ISSUE TITLE
Warning when running importer

### ISSUE THREAD

**Issue by omerlh** [snyk/snyk-github-import#5]
I tried to use the importer, it works but logs a warning:
```
Importing repos modified since 2018-12-16T00:00:00+00:00
DEPRECATED (@octokit/rest): `repos.getAll()` is deprecated, use `repos.list()`
Found 11 repos to import
```

**Comment by joshje** [comment]
Thanks for bringing this to our attention. We will release a fix soon!

**Comment by joshje** [comment]
Now resolved in [v1.2.0](https://github.com/snyk/snyk-github-import/releases/tag/v1.2.0).

**Comment by omerlh** [comment]
Yay! Thanks :)
